airflow/providers/cncf/kubernetes/operators/custom_object_launcher.py

```diff
@@ -0,0 +1,333 @@
+            executor['coreLimit'] = self.executor['cpu']['limit']
+        if self.executor['memory'].get('limit'):
+            executor['memory'] = self.executor['memory']['limit']
+        if self.executor['gpu'].get('name') and self.executor['gpu'].get('quantity'):
+            executor['gpu'] = {'name': self.executor['gpu']['name'], 'quantity': self.executor['gpu']['quantity']}
+        return executor
+
+    def convert_resources(self):
+        if isinstance(self.driver['memory'].get('limit'), str):
+            if 'G' in self.driver['memory']['limit'] or 'Gi' in self.driver['memory']['limit']:
+                self.driver['memory']['limit'] = float(self.driver['memory']['limit'].rstrip('Gi G')) * 1024
+            elif 'm' in self.driver['memory']['limit']:
+                self.driver['memory']['limit'] = float(self.driver['memory']['limit'].rstrip('m'))
+            # Adjusting the memory value as operator adds 40% to the given value
+            self.driver['memory']['limit'] = str(int(self.driver['memory']['limit'] / 1.4)) + 'm'
+
+        if isinstance(self.executor['memory'].get('limit'), str):
+            if 'G' in self.executor['memory']['limit'] or 'Gi' in self.executor['memory']['limit']:
+                self.executor['memory']['limit'] = float(self.executor['memory']['limit'].rstrip('Gi G')) * 1024
+            elif 'm' in self.executor['memory']['limit']:
+                self.executor['memory']['limit'] = float(self.executor['memory']['limit'].rstrip('m'))
+            # Adjusting the memory value as operator adds 40% to the given value
+            self.executor['memory']['limit'] = str(int(self.executor['memory']['limit'] / 1.4)) + 'm'
+
+        if self.driver['cpu'].get('request'):
+            self.driver['cpu']['request'] = int(float(self.driver['cpu']['request']))
+        if self.driver['cpu'].get('limit'):
+            self.driver['cpu']['limit'] = str(self.driver['cpu']['limit'])
+        if self.executor['cpu'].get('request'):
+            self.executor['cpu']['request'] = int(float(self.executor['cpu']['request']))
+        if self.executor['cpu'].get('limit'):
+            self.executor['cpu']['limit'] = str(self.executor['cpu']['limit'])
+
+        if self.driver['gpu'].get('quantity'):
+            self.driver['gpu']['quantity'] = int(float(driver['gpu']['quantity']))
+        if self.executor['gpu'].get('quantity'):
+            self.executor['gpu']['quantity'] = int(float(self.executor['gpu']['quantity']))
+
+
+class CustomObjectStatus:
+    """Status of the PODs"""
+
+    SUBMITTED = 'SUBMITTED'
+    RUNNING = 'RUNNING'
+    FAILED = 'FAILED'
+    SUCCEEDED = 'SUCCEEDED'
+
+
+class CustomObjectLauncher(LoggingMixin):
+    """Launches PODS"""
+
+    def __init__(
+        self,
+        name: str,
+        namespace: str,
+        kube_client: client.CoreV1Api,
+        custom_obj_api: client.CustomObjectsApi,
+        template_body: Optional[str] = None,
+    ):
+        """
+        Creates the launcher.
```

Comment:
Which launcher? Maybe we should specify?

Reply:
done in [228ce7e](https://github.com/apache/airflow/pull/22253/commits/228ce7ee602132e51544032d23fc47f5720177a1)